widget/ui/src/components/Drawer/Drawer.tsx

```diff
@@ -25,7 +25,7 @@ export function Drawer(props: PropsWithChildren<PropTypes>) {
     anchor = 'bottom',
     dismissible = true,
     footer,
-    container = document.body,
+    container = document.getElementById('swap-box'),
```

Comment:
The drawer does not work in the storybook. Because you changed the default container

Reply:
it works, I'd already returned it to the main version.